CSharp/DevVmPowershell/Helpers/Implementations/AgentHelper.cs

```diff
@@ -86,66 +86,105 @@ private async Task<List<int>> GetAgentArtifactIdsAsync(string agentName)
 			return agentArtifactIds;
 		}
 
-		private async Task<List<AgentTypeResponse>> GetAgentTypesInInstanceAsync()
+		private async Task<List<AgentTypeResponseModel>> GetAgentTypesInInstanceAsync()
 		{
-			using (Relativity.Services.Interfaces.Agent.IAgentManager agentManager = ServiceFactory.CreateProxy<Relativity.Services.Interfaces.Agent.IAgentManager>())
+			HttpClient httpClient = RestHelper.GetHttpClient(InstanceAddress, AdminUsername, AdminPassword);
+			HttpResponseMessage response = await RestHelper.MakeGetAsync(httpClient, Constants.Connection.RestUrlEndpoints.AgentType.EndpointUrl);
+			if (!response.IsSuccessStatusCode)
 			{
-				List<AgentTypeResponse> agentTypeResponseList = await agentManager.GetAgentTypesAsync(Constants.EDDS_WORKSPACE_ARTIFACT_ID);
-				Console.WriteLine($"Total Agent Types in Instance: {agentTypeResponseList.Count}");
-				return agentTypeResponseList;
+				throw new Exception("Failed to Get Agent Types in Instance");
 			}
+			string resultString = await response.Content.ReadAsStringAsync();
+			dynamic result = JsonConvert.DeserializeObject<dynamic>(resultString);
+			List<AgentTypeResponseModel> agentTypeResponseList = new List<AgentTypeResponseModel>();
+			foreach (dynamic obj in result)
+			{
+				agentTypeResponseList.Add(new AgentTypeResponseModel()
+				{
+					ApplicationName = obj["ApplicationName"].ToString(),
+					CompanyName = obj["CompanyName"].ToString(),
+					DefaultInterval = Convert.ToDecimal(obj["DefaultInterval"].ToString()),
+					DefaultLoggingLevel = Convert.ToInt32(obj["DefaultLoggingLevel"].ToString()),
+					ArtifactID = Convert.ToInt32(obj["ArtifactID"].ToString()),
+					Name = obj["Name"].ToString()
+				});
+			}
+
+			return agentTypeResponseList;
 		}
 
-		private async Task<List<AgentServerResponse>> GetAgentServersForAgentTypeAsync(int agentTypeArtifactId)
+		private async Task<List<AgentServerResponseModel>> GetAgentServersForAgentTypeAsync(int agentTypeArtifactId)
 		{
-			using (Relativity.Services.Interfaces.Agent.IAgentManager agentManager = ServiceFactory.CreateProxy<Relativity.Services.Interfaces.Agent.IAgentManager>())
+			string url = $"Relativity.rest/api/relativity.agents/workspace/-1/agenttypes/{agentTypeArtifactId}/availableagentservers";

```

Comment:
- [ ] Move the url to the Constants class

Reply:
Done